sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,282 @@
+- Updated [`sycl_ext_oneapi_accessor_properties`](sycl/doc/extensions/supported/sycl_ext_oneapi_accessor_properties.asciidoc)
+  to mark `has_property` API as `noexcept`. [7805aa3f]
+- Updated [`sycl_ext_intel_device_info`](sycl/doc/extensions/supported/sycl_ext_intel_device_info.md)
+  to support querying free device memory. [0eeef2b3]
+- Updated [`sycl_ext_oneapi_matrix`](sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix.asciidoc)
+  with description of new matrix features. [770f540d]
+- Moved [`sycl_ext_oneapi_invoke_simd`](sycl/doc/extensions/experimental/sycl_ext_oneapi_invoke_simd.asciidoc)
+  extensions specification from `proposed` to `experimental` because
+  implementation is available. [6bee3440]
+
+## Improvements
+
+### SYCL Library
+
+- Ensured that a correct errc thrown for unassociated placeholder accessor (4.7.6.9) [4f9935ad]
+- Removed dependency on OpenCL ICD Loader from the runtime [90e8b5ef]
+- Added `group::get_linear_id(int)` method [6e83c127]
+- Added support for `ZEBIN` format to persistent caching mechanism [34dcf83d]
+- Added identification mechanism for binaries in newer `ZEBIN` format. [f4dee549]
+- Fixed uses of common macro names in the imlementation's header files [e87adfd2]
+- Switched to use `struct` information descriptors in accordance with SYCL 2020.
+  Removed some deprecated infromation queries. [b3cbda57]
+- Updated `kernel_device_specific::max_sub_group_size` query to match SYCL 2020
+  spec. Deprecated the old variant. [7842d056]
+- Deprecated SYCL 1.2.1 device selectors. [c058380f]
+- Improved error messages reported for unsupported devie partitioning. [1c9ddbaa]
+- Made `device` and `platform` default to `default_selector_v`. [b32dd41a]
+- Deprecated `address_space::constant_space`. [351b123a]
+- Marked `sycl::exception::has_context` as `noexcept`. [ad923c9a]
+- Improved range reductions performance on CPU. [3323da65]
+- Made `sycl::exception` `nothrow` copy constructible. [289e33da]
+- Marked `has_property` methods as `noexcept`. [417b5a29]
+- Improved `sycl::event::get_profiling_info` exception message when `event` is
+  default constructed. [2e86cd459]
+- Added a diagnostic (in form of `static_assert`) about kernel lambda size
+  mismatch between host and device. [d278c671] [ec179b7a] [f417a88e]
+- Updated `pipes` class to throw exceptions if used on host. [eab29696]
+- Updated ESIMD Emulator PI plugin to report support for `cl_khr_fp64`
+  extension. [398571a5]
+- Updated Level Zero plugin to prefer copy engine for memory read/write
+  operations. [65c3ea29]
+- Optimized some memory transfers. [92d35cd1]
+- Enabled event caching in Level Zero PI plugin. [a41b33c3]
+- Optimized some reductions for discrete GPUs. [c22a5d3f]
+- Improved performance of event synchronization on CUDA backend. [c4f326aa]
+- Added ability to use descendent devices of context members can now be used
```

Comment:
```suggestion
- Added ability to use descendent devices of context members
```

Reply:
Applied in ed5e71d464608b7be8fd28ee14249979e67e4fb8